doc/kernel/services/scheduling/index.rst

```diff
@@ -82,7 +82,11 @@ runtime overhead and performance scaling when many threads are added.
 * Traditional multi-queue ready queue (:kconfig:option:`CONFIG_SCHED_MULTIQ`)
 
   When selected, the scheduler ready queue will be implemented as the
-  classic/textbook array of lists, one per priority (max 32 priorities).
+  classic/textbook array of lists, one per priority.
+
+  This scheduler will invoke mod operation so it is suggested
+  to set (:kconfig:option:`CONFIG_NUM_PREEMPT_PRIORITIES`) + (:kconfig:option:`CONFIG_NUM_COOP_PRIORITIES`)
+  to power of 2 minus 1 (e.g. 31, 63, 127...).
```

Comment:
It should be fine to set either `CONFIG_NUM_PREEMPT_PRIORITIES` or `CONFIG_NUM_COOP_PRIORITIES` to a value that is not a power of two minus one as well.

From what I understand, there can be a maximum of 256 priorities in total (including both preempt and coop) based on this

https://github.com/zephyrproject-rtos/zephyr/blob/840db7858e90cb49c1ba253ce037a44c01549b18/include/zephyr/kernel/thread.h#L95

I'm not sure if we need more than 256, to be honest.

Maybe this could be more short and sweet?
```suggestion
  classic/textbook array of lists, one per priority.
```